### PR TITLE
Telemetry changes

### PR DESCRIPTION
**Pull request by sathishkumarsivagurunathan**
Adding the telemetry changes

### CODE REVIEW

File: hpe3parclient/client.py [hunk 1]
```diff
@@ -491,7 +491,7 @@ def createVolume(self, name, cpgName, sizeMiB, optional=None):
             - EXISTENT_SV - Volume Exists already
 
         """
-        info = {'name': name, 'cpg': cpgName, 'sizeMiB': sizeMiB}
+        info = {'name': name, 'cpg': cpgName, 'sizeMiB': sizeMiB, 'objectKeyValues': [{'key': 'type', 'value': self.app_type}]}
```
Comment (wdurairaj) [inline hpe3parclient/client.py]: Ideally the objectKeyValues map should come as part of optional arguments to the createVolume call. Use this approach from the caller library.

Comment (wdurairaj) [reply hpe3parclient/client.py]: python-3parclient should'nt be changed for this kind of changes.

Comment (hemna) [reply hpe3parclient/client.py]: Please do NOT make the default client debug level ON!!!!
It's a major security risk as well to enable SSL warnings by default.  Yes, they are annoying, but they are there for a reason.   Always always always error on the side of security.

**Comment by hemna** [comment]
These changes are going to break backwards compatibility with users of the client.  You can't change the default behavior.

**Pull request closed without merging**